<launch>

	<rosparam command="load" file="$(env HOME)/workspace/data/bags/stereo/zed_hd/cam0.yaml" ns="/cam0"/>
	<rosparam command="load" file="$(env HOME)/workspace/data/bags/stereo/zed_hd/cam1.yaml" ns="/cam1"/>

	 <node name="manager_node" pkg="direct_stereo" type="manager_node" output="screen" >
	    <param name="cam0_topic" value="/zed/left/image_raw_color" />
	    <param name="cam1_topic" value="/zed/right/image_raw_color" />
	    <param name="gt_type" value="odom" />
	    <param name="cvt2VGA" value="false" />
    </node>

</launch>
